x-pack/legacy/plugins/apm/public/components/app/ErrorGroupDetails/DetailView/index.tsx

```diff
@@ -94,7 +110,34 @@ export function DetailView({ errorGroup, urlParams, location }: Props) {
         </DiscoverErrorLink>
       </HeaderContainer>
 
-      <StickyErrorProperties error={error} transaction={transaction} />
+      <Summary
+        items={
+          [
+            <TimestampSummaryItem time={error.timestamp.us / 1000} />,
+            errorUrl && method ? (
+              <HttpInfoSummaryItem
+                url={errorUrl}
+                method={method}
+                status={status}
+              />
+            ) : null,
+            transaction && (
+              <TransactionDetailLink
+                traceId={transaction.trace.id}
+                transactionId={transaction.transaction.id}
+                transactionName={transaction.transaction.name}
+                transactionType={transaction.transaction.type}
+                serviceName={transaction.service.name}
+              >
+                <EuiIcon type="merge" />
+                <TransactionLinkName>
+                  {transaction.transaction.name}
+                </TransactionLinkName>
+              </TransactionDetailLink>
+            )
+          ].filter(Boolean) as React.ReactElement[]
```

Comment:
Perhaps move this into `Summary/index.tsx` and change the type to:
```
items: Array<React.ReactElement | undefined>;
```

Reply:
Fixed.